setup.py

```diff
@@ -6,12 +6,12 @@
 install_requires = [line.rstrip() for line in open(os.path.join(os.path.dirname(__file__), "requirements.txt"))]
 
 setup(
-    name="dbio",
+    name="dbio-cli",
```

Comment:
dumb q?  is this gonna change the command from `dbio` to `dbio-cli`? 🤔   or is this only the pypi app?

Reply:
I think the actual command line utility's name is specified elsewhere, but this is a good question. Let me look into that and confirm that this won't change the CLI tool's name.

Reply:
Line 27 (https://github.com/DataBiosphere/data-store-cli/pull/56/files#diff-2eeaed663bd0d25b7e608891384b7298R27) of setup.py is where the CLI tool gets its name: 

```
entry_points={
        'console_scripts': [
            'dbio=dbio.cli:main'
        ],
    },
```

There, we specify that the command `dbio` (that's where the name of the tool is specified) should map to the python file at `dbio/cli.py`, and the function `main()`.